benches/src/bin/collect.rs

```diff
@@ -642,7 +642,7 @@ fn linear_regression(x_y: Vec<(u64, u64)>) -> f64 {
 }
 
 fn dependent_cost(name: &String, x_y: Vec<(u64, u64)>) -> DependentCost {
-    const NEAR_LINEAR: f64 = 0.1;
+    const NEAR_LINEAR: f64 = 0.2;
```

Comment:
Shall we also change the comment in [this line](https://github.com/FuelLabs/fuel-core/pull/2153/files#diff-4ce720017e878bdefcfe9d40e80692be37fb307f5fcff3b12d271c6dad0ed694R650) to:
```
/// and the last points are almost the same(The difference is < 0.2).
```
or maybe even
```
/// and the last points are almost the same(The difference is < NEAR_LINEAR).
```